gui/templates/rebalances_table.html

```diff
@@ -1,115 +1,144 @@
-    transformations = {
-      'requested': rebalance => ({innerHTML: formatDate(rebalance.requested), title: adjustTZ(rebalance.requested)}),
-      'start': rebalance => ({innerHTML: formatDate(rebalance.start), title: rebalance.start ? adjustTZ(rebalance.start) : '---'}),
-      'stop': rebalance => ({innerHTML: formatDate(rebalance.stop), title: rebalance.stop ? adjustTZ(rebalance.stop) : '---'}),
-      'duration': rebalance => ({innerHTML: formatDuration(rebalance.duration)}),
-      'elapsed': rebalance => ({innerHTML: formatDate(rebalance.start, rebalance.stop).replace(" ago", "")}),
-      'value': rebalance => ({innerHTML: Number(rebalance.value).toLocaleString()}),
-      'fee_limit': rebalance => ({innerHTML: Number(rebalance.fee_limit).toLocaleString()}),
-      'ppm': rebalance => ({innerHTML: Math.round(rebalance.fee_limit*1000000/rebalance.value).toLocaleString()}),
-      'fees_paid': rebalance => ({innerHTML: rebalance.status == 2 ? Number(rebalance.fees_paid).toLocaleString() : '---'}),
-      'target_alias': rebalance => ({innerHTML: rebalance.target_alias !== '' ? rebalance.target_alias : '---'}),
-      'status': rebalance => ({innerHTML: statuses[rebalance.status], title: rebalance.status}),
-      'payment_hash': rebalance => ({innerHTML: (rebalance.payment_hash || '').length == 0 ? '---' :
-           `<a href="/route?=${rebalance.payment_hash}" target="_blank">${rebalance.payment_hash.substring(0,7)}</a>`}),
-      'action': rebalance => {
-        const input = document.createElement("button")
-        if (rebalance.status != 0){
-          input.innerHTML = "🔂"
-          input.title="Repeat this request" 
-          input.onclick = async function(){
-            const {value, fee_limit, outgoing_chan_ids, last_hop_pubkey, duration, target_alias, manual} = rebalance
-            if (last_hop_pubkey.length === 0){
-              var new_rebal = {value, fee_limit, outgoing_chan_ids, duration, manual}
-            }else{
-              var new_rebal = {value, fee_limit, outgoing_chan_ids, last_hop_pubkey, duration, target_alias, manual}
-            }
-            const response = await POST('rebalancer', {body: new_rebal})
-            const table = input.parentElement.parentElement.parentElement
-            table.prepend(use(transformations).render(response))
-          } 
-        }
-        else {
-          input.innerHTML = "❌"
-          input.title="Cancel this request" 
-          input.onclick = async function(){
-            const response = await PUT(`rebalancer/${rebalance.id}`, {body: {status: 499}})
-            
-            const table = input.parentElement.parentElement.parentElement
-            const index = input.parentElement.parentElement.rowIndex -1
-            table.deleteRow(index)
-            use(transformations).render(response, table.insertRow(index))
+    </tfoot>
+  </table>
+</div>
+<script>
+  var rapidFire = 'rapidFire', target_time = {{target_time}}
```

Comment:
I see `target_time` is passed in and stored here as a variable but where is it used after that?